src/ol/source/wmtssource.js

```diff
@@ -428,6 +401,19 @@ ol.source.WMTS.optionsFromCapabilities = function(wmtsCap, config) {
         /urn:ogc:def:crs:(\w+):(.*:)?(\w+)$/, '$1:$3'));
   }
 
+  var projectionExtent = projection.getExtent();
+  var extent;
+  if (!goog.isNull(projectionExtent)) {
```

Comment:
Right, I will add a FIXME here. Currently our WMTS capabilities format parser only reads the WGS84BoundingBox of the layer and not the bounding box in different CRSs. The reason for this logic is that the WGS84BoundingBox cannot always be transformed to the data projection, and we need to safeguard against that. A cleaner solution will be to use the advertised BBOX in the data projection.